src/helpers/config.ts

```diff
@@ -13,6 +14,7 @@ const config: AuthConfig = {
   client_secret: import.meta.env.VITE_CLIENT_SECRET,
   scope: import.meta.env.VITE_SCOPE,
   authority: import.meta.env.VITE_OIDC_AUTHORITY,
+  cognito_logout_uri: import.meta.env.VITE_COGNITO_LOGOUT_URI,
```

Comment:
Setting up this is fine. But until cognito is released, this should work with CAS.